src/SendPacketZMQ.h

```diff
@@ -20,3 +12,17 @@ class SendPacketZMQ : public Rifle
    void SetQueueSize(const int size);
    bool Initialize();
 };
+
+
+/*#include <dpi/libafc.h>
```

Comment:
Thanks.   

I'll leave this in until I have the Research pull request up.  I'll remove it before merging.